docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1072,212 +1072,602 @@ thor:
 
-          <para>Some of those variables are available for container and cloud
-          deployments and can be set using the Helm chart. The following
-          bare-metal environment.conf values have these equivalent values
-          which can be set for containerized instances.</para>
+          <listitem>
+            <para>Pod topology spread constraints</para>
+          </listitem>
+        </itemizedlist></para>
 
-          <para><informaltable>
-              <tgroup cols="2">
-                <tbody>
-                  <row>
-                    <entry><emphasis role="bold">Environment.conf
-                    Value</emphasis></entry>
+      <sect3 id="CV_NodeLabels">
+        <title>Node Labels</title>
 
-                    <entry><emphasis role="bold">Helm Environment
-                    Variable</emphasis></entry>
-                  </row>
+        <para>Kubernetes nodes have labels. Kubernetes has a standard set of
+        labels used for nodes in a cluster. You can also manually attach
+        labels which is recommended as the value of these labels is
+        cloud-provider specific and not guaranteed to be reliable.</para>
 
-                  <row>
-                    <entry>skipPythonCleanup</entry>
+        <para>Adding labels to nodes allows you to schedule pods to nodes or
+        groups of nodes. You can then use this functionality to ensure that
+        specific pods only run on nodes with certain properties.</para>
+      </sect3>
 
-                    <entry>SKIP_PYTHON_CLEANUP</entry>
-                  </row>
+      <sect3 id="CV_nodeSelector">
+        <title>The nodeSelector</title>
+
+        <para>The nodeSelector is a field in the Pod specification that allows
+        you to specify a set of node labels that must be present on the target
+        node for the Pod to be scheduled there. It is the simplest form of
+        node selection constraint. It selects nodes based on the labels, but
+        it has some limitations. It only supports one label key and one label
+        value. If you wanted to match multiple labels or use more complex
+        expressions, you need to use node Affinity.</para>
+
+        <para>Add the nodeSelector field to your pod specification and specify
+        the node labels you want the target node to have. You must have the
+        node labels defined in the job and pod. Then you need to specify each
+        node group the node label to use. Kubernetes only schedules the pod
+        onto nodes that have the labels you specify.</para>
+
+        <para>The following example shows the nodeSelector placed in the pods
+        list. This example schedules "all" HPCC components to use the node
+        pool with the label group: "hpcc".</para>
+
+        <para><programlisting> placements:
+   - pods: ["all"]
+     placement:
+       nodeSelector:
+         group: "hpcc"</programlisting></para>
+
+        <para>Note:the label: group:hpcc matches the node pool label:
```

Comment:
Note: s/b bold, then a space and capitalize the sentence. also no need for the quotes around hpcc. 

**Note:** The label: group:hpcc matches the node pool label:hpcc.